File: Bank/EntitiesCs/SavingAccount.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EntitiesCs
{
    public class SavingAccount : Account
    {
        public SavingAccount(int number, decimal balance, decimal monthlyInterestRate) : base(number, balance)
        {
            MonthlyInterestRate = monthlyInterestRate;
        }
        public decimal MonthlyInterestRate { get; set; }
        public void DepositMonthlyInterest()
        {
            Deposit(Balance * MonthlyInterestRate / 100);
        }
    }
}

File: Bank/EntitiesVb/SavingAccount.vb
Public Class SavingAccount
    Inherits Account
    Public Sub New(number As Integer, balance As Decimal, monthlyInterestRate As Decimal)
        MyBase.New(number, balance)
        Me.MonthlyInterestRate = monthlyInterestRate
    End Sub
    Public Property MonthlyInterestRate As Decimal
    Public Sub DepositMonthlyInterest()
        Deposit(Balance * MonthlyInterestRate / 100)
    End Sub
End Class
